### DISCUSSION TITLE
HW_01 Question

### DISCUSSION THREAD

**Discussion by travismalave** [cooperrc/advanced-dynamics#15] [General]
Hi All,

Having a little trouble getting started with HW_01 and was wondering if anyone has some feedback on a starting direction. I was reviewing the lectures and text, but I can't seem to find the right information to get me going. I understand that the angular position would be the initial position plus the equation of motion of the pendulum but can't seem to isolate this equation (general harmonic equation comes to mind but I don't believe its appropriate here). Appreciate any feedback or info to help!

**Comment by SentientBiped** [comment]
Go back to a free-body diagram of the mass on the pendulum but now with this external acceleration present. You should be able to derive an equation for theta under these conditions in the same way you would for the case with only gravity. [https://www.acs.psu.edu/drussell/Demos/Pendulum/Pendulum.html](url)

**Reply by travismalave** [reply]
Thanks for the reference! I came across this too. This is where my thoughts on a general harmonic equation came from. The reference does provide an expression in the form of F=ma from the lecture, so I'll take another look at it and see what I can do from there. Thanks again!

**Comment by RebeccaVillanueva** [comment]
Hello, while I was refreshing myself on this stuff I found this youtube video to be helpful! https://www.youtube.com/watch?v=MdECjGYd7n0

edit: idk why the link doesn't look like a link but hopefully you can copy and paste

**Reply by SentientBiped** [reply]
Cuh ain't that a static solution, though?

**Reply by travismalave** [reply]
Appreciate the additional reference for my question above!

The YouTube video seems to be using an example of a pendulum on an accelerating/moving reference so it may offer support on this HW question (and mine). I haven't watched the whole video yet though.

**Comment by tingkai-li** [comment]
I wonder if there was a typo in part (b) of HW1. Is $X_2$ in the problem statement $Y_1$ in Fig. 1-5?
